pom.xml

```diff
@@ -1278,15 +1285,15 @@
             <dependency>
               <groupId>org.apache.maven.surefire</groupId>
               <artifactId>surefire-junit47</artifactId>
-              <version>2.19.1</version>
+              <version>2.20</version>
             </dependency>
           </dependencies>
         </plugin>
 
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-failsafe-plugin</artifactId>
-          <version>2.19.1</version>
+          <version>2.20</version>
```

Comment:
Why is this needed in this PR? If not, please upgrade deps in separate PR from refactoring.